### ISSUE TITLE
Studio/Project Problem

### ISSUE THREAD

**Issue by leahcimto** [LLK/scratch-www#4674]
Hi, I found a problem with the "Add to Studio" button. When you press the button to remove a project from a studio, but then press the close button (not the okay button) and refresh the page, the project will be removed. 
![Scratch-Studio-Bug](https://user-images.githubusercontent.com/68165163/99890543-69cb9700-2c2e-11eb-9e0f-62b5f3dde303.gif)

### Steps to Reproduce

1. Create a new project and share it
2. On another account, make a studio and add the project you just shared to it
3. Back on the first account, press "Add to Studio"
4. Press the checkmark with the name of the studio you made but do not press okay
5. Press the "Close"  button
6. Refresh the page. You will notice that the project was removed from the studio, even though you never pressed "Okay".

### Operating System and Browser
Windows 10, Chrome  86.0.4240.198

(Sorry if I am doing something wrong, this is my first time doing this)

**Comment by leahcimto** [comment]
tl;dr: When pressing the close button on the "Add to Studio" menu, it does the same thing as the "Okay" button after the page is refreshed.

**Comment by BryceLTaylor** [comment]
We should remove one of the buttons.  Right now we're leaning towards just having the Close button.

**Comment by leahcimto** [comment]
Same with the x button.
